src/script/components/MessagesList/MessageList.tsx

```diff
@@ -250,6 +250,7 @@ const MessagesList: FC<MessagesListParams> = ({
         onLoading(false);
       }, 10);
     });
+    return () => conversation.release();
```

Comment:
We just need to release the conversation when the message list is being unloaded or the conversation changes :)